R/query-set-op.R

```diff
@@ -24,16 +24,18 @@ print.set_op_query <- function(x, ...) {
 }
 
 #' @export
-sql_render.set_op_query <- function(query, con = NULL, ..., subquery = FALSE) {
-  from_x <- sql_render(query$x, con, ..., subquery = FALSE)
-  from_y <- sql_render(query$y, con, ..., subquery = FALSE)
+sql_render.set_op_query <- function(query, con = NULL, ..., subquery = FALSE, lvl = 0) {
+  # TODO improve this hack a little
+  sub_lvl <- lvl + !inherits(con, "SQLiteConnection")
```

Comment:
This is necessary because `sql_query_set_op.DBIConnection()` wraps the queries in parantheses whereas `sql_query_set_op.SQLiteConnection()` does not.